test/connectSpec.js

```diff
@@ -519,6 +528,20 @@ describe('Connection methods', function () {
       return assert.isRejected(this.plugin.connect(),
         /Error: Failed to resolve ledger URI from account URI/)
     })
+
+    it('fails if the retries exceed the timeout', function () {
+      const plugin = new PluginBells({
+        prefix: 'example.red.',
+        account: 'http://red.example/accounts/mike',
+        password: 'mike',
+        connectTimeout: 1000
```

Comment:
Unfortunately, fake timers don't appear to play well with websockets -- as soon as I enable the fake timers, the websocket won't receive any messages.